### DISCUSSION TITLE
CSS not working properly

### DISCUSSION THREAD

**Discussion by rrenildopereiraa** [vercel/community#5594] [Help] [answered]
### Summary

I've been experiencing some weird CSS inconsistencies in my Docusaurus while using Vercel.

### Example

https://yummacss.vercel.app/

### Steps to Reproduce

When trying to access my index.js page I see that most of my styles are all broken. I've tried modifying and redeploying without cache and nothing solves my problem.

![localhost](https://github.com/vercel/community/assets/56491937/72250c7d-9ddf-471e-8f69-254896012718)
![production](https://github.com/vercel/community/assets/56491937/cd10918d-b3f8-4dad-a260-e001554d622f)

**Comment by noahbazer** [comment]
Hey! Could you provide more information about you setup (browser, browser plugins, etc)? From what I can tell, all of the styles are displaying correctly on my end. (Looks great, by the way!)
![image_2024-01-31_003520791](https://github.com/vercel/community/assets/125102673/69f4f2f4-f2e1-4794-ae4b-8d229a6d1dc8)

**Comment by rrenildopereiraa** [comment, marked as answer]
I’ve fixed the issues… I was using SASS nesting syntax accidentally instead of CSS…

Renildo
Sent with Spark
